base_tier_validation_report/reports/tier_review_report.xml

```diff
@@ -0,0 +1,151 @@
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="resource_type" />
+                            <field name="resource_name" />
+                            <field name="requested_by" />
+                            <field name="review_type" />
+                            <field name="name" />
+                        </group>
+                        <group>
+                            <field name="todo_by" />
+                            <field name="done_by" />
+                            <field name="reviewed_date" />
+                            <field name="comment" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="tier_review_report_search" model="ir.ui.view">
+        <field name="name">tier.review.report.search</field>
+        <field name="model">tier.review</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="resource_ref" string="Resource Ref" />
+                <field name="requested_by" string="Requested By" />
+                <field name="reviewer_ids" string="To Review" />
+                <field name="done_by" string="Done By" />
+                <filter
+                    name="pending"
+                    string="To Review"
+                    domain="[('status', '=', 'pending')]"
+                />
+                <filter
+                    name="rejected"
+                    string="Rejected"
+                    domain="[('status', '=', 'rejected')]"
+                />
+                <filter
+                    name="approved"
+                    string="Approved"
+                    domain="[('status', '=', 'approved')]"
+                />
+            </search>
+        </field>
+    </record>
+
+    <record id="tier_review_my_action" model="ir.actions.act_window">
+        <field name="name">My Reviews</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">tier.review</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[("reviewer_ids", "in", uid)]</field>
+        <field name="context">{"search_default_pending": True}</field>
+        <field name="view_id" ref="tier_review_report_tree" />
+        <field name="search_view_id" ref="tier_review_report_search" />
+    </record>
+
+    <record id="tier_review_all_action" model="ir.actions.act_window">
+        <field name="name">All Reviews</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">tier.review</field>
+        <field name="view_mode">tree,form</field>
+        <field name="context">{}</field>
+        <field name="view_id" ref="tier_review_report_tree" />
+        <field name="search_view_id" ref="tier_review_report_search" />
+        <field
```

Comment:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

Any view with groups_id on it now has to move such groups to the elements of the view. But now you can put groups attribute in a view field instead of isolating it on a view without fear of an access error.